android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/BridgeManagerProvider.java

```diff
@@ -113,6 +117,8 @@ private BridgeManagerProvider(@NonNull Context applicationContext) {
     @NonNull
     private final AccountDAO accountDAO;
     @NonNull
+    private final UploadDAO uploadDAO;
```

Comment:
I think that should be a part of this story.  Seems like a really easy task, no?  On successful sign out, call clear.